# 0: unlocked, 1: locked
uint8 pps_state

#  0: "V" invalid, 1:"A" received, 2: unkonwn
uint8 nmea_state

uint32 gps_timestamp